src/systemctl/systemctl.c

```diff
@@ -9063,19 +9064,24 @@ static int logind_schedule_shutdown(void) {
         switch (arg_action) {
         case ACTION_HALT:
                 action = "halt";
+                log_action = "Shutdown";
                 break;
         case ACTION_POWEROFF:
                 action = "poweroff";
+                log_action = "Shutdown";
                 break;
         case ACTION_KEXEC:
                 action = "kexec";
+                log_action = "Kernel replacement";
```

Comment:
Please change this to "Reboot via kexec". "Kernel replacement" is confusing.

Reply:
@keszybz isn't that *even more* confusing? Because as far as I understand *no reboot* will happen, it's only the kernel that will be replaced.

I can replace it with "Kernel replacement via kexec" (...scheduled for Thu 2020-06-11 06:00:00 CEST, use 'shutdown -c' to cancel."). Or the shorter, much less explicit and IMHO much less clear "Kexec" (...scheduled for Thu 2020-06-11 06:00:00 CEST, use 'shutdown -c' to cancel.").

?

Reply:
From a users pov, it is a new boot. The difference is that the system doesn't go through POST or similar and the hardware isn't physically reset. But depending on how the machine is set up, this might not even make any visual difference. The system starts from the beginning.

I'm pretty sure that "boot"/"reboot" and "kexec" should appear somewhere in this string.
Maybe "Boot into new kernel with kexec" ?

Reply:
Ah, I see. So it was a misunderstanding on my side. Will change accordingly. Thanks!